content/articles/creating-a-hangman-game-with-vanilla-javascript/index.md

```diff
@@ -0,0 +1,297 @@
+### Creating a Hangman game with Vanilla JavaScript
+The Hangman game is a word-guessing game. One player picks a secret word, and the other player tries to guess it. In this article, I will take you through building a simple Hangman game with just vanilla JavaScript, and more importantly, how I made it accessible to players who navigate with their keyboards, without using any frameworks.
+
+### Prerequisites
+1. A good code editor. Visual Studio Code can do the job.
+1. Some knowledge of HTML, Bootstrap, and JavaScript.
+
+### Designing the game
+The first thing you have to do is create three files on your code editor and name them `index.html` for your HTML code and a `script.js` for your JavaScript code. I mostly used Bootstrap CDN for styling this game. You can add the Bootstrap CDN by adding the code below inside the `<head></head>` of your HTML file.
+
+```html
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+```
+
+Below is the entire HTML code. The images used in this project can be found in my [Github repo](https://github.com/Nomzy-kush/Hangman-Game).
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+
+  <title>Hangman Game with JavaScript</title>
+</head>
+<body>
+<div class="container">
+  <h1 class="text-center" style="padding-top: 2em;">Hangman</h1>
+  <div class="float-right">Wrong Guesses: <span id='mistakes'>0</span> of <span id='maxWrong'></span></div>
+  <div class="text-center">
+    <img id='hangmanPic' src="images/0.jpg" alt="">
+    <p>Guess the Social Medium:</p>
+    <p id="wordSpotlight">The word to be guessed goes here</p>
+    <div id="keyboard"></div>
+    <button class="btn btn-info" onClick="reset()">Reset</button>
+  </div>
+</div>
+
+<script src='script.js'></script>
+</body>
+</html>
+```
+
+### The Actual JavaScript
+First, you need to link your Javascript code to your HTML file by pasting the code below at the end of the `<body></body>` tag.
+
+```html
+<script src='script.js'></script>
+```
+
```

Comment:
This line is already in the previous block. Please remove it to avoid repetitio.